src/lib.rs

```diff
@@ -841,23 +936,32 @@ impl EGraph {
                     "Skipping schedule.".to_string()
                 }
             }
-            NCommand::Extract { var, variants } => {
+            /*NCommand::Extract { var, variants } => {
```

Comment:
Same question here: I think it'd help if large commented-out blocks like this should either be removed or documented a bit more.